vmm/sandbox/src/lib.rs

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2022 The Kuasar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use std::{env, path::Path};
+
+use anyhow::Result;
+use cloud_hypervisor::{
+    config::CloudHypervisorVMConfig, factory::CloudHypervisorVMFactory, hooks::CloudHypervisorHooks,
+};
+use kata_config::KataConfig;
+use qemu::{factory::QemuVMFactory, hooks::QemuHooks};
+use sandbox::KuasarSandboxer;
+use serde::de::DeserializeOwned;
+use stratovirt::{
+    config::StratoVirtVMConfig, factory::StratoVirtVMFactory, hooks::StratoVirtHooks,
+};
+
+use crate::{config::Config, sandbox::KuasarSandbox};
+
+#[macro_use]
+mod device;
+
+mod cgroup;
+mod client;
+mod container;
+mod io;
+mod network;
+mod param;
+mod storage;
+mod utils;
+mod vm;
+
+pub mod cloud_hypervisor;
+pub mod config;
+pub mod kata_config;
+pub mod qemu;
+pub mod sandbox;
+pub mod stratovirt;
+
+pub const NAMESPACE_PID: &str = "pid";
+pub const NAMESPACE_NET: &str = "network";
+pub const NAMESPACE_MNT: &str = "mount";
+pub const NAMESPACE_CGROUP: &str = "cgroup";
+
+pub const FS_SHARE_PATH: &str = "shared_fs";
+
+pub const CONFIG_STRATOVIRT_PATH: &str = "/var/lib/kuasar/config_stratovirt.toml";
+pub const CONFIG_CLH_PATH: &str = "/var/lib/kuasar/config_clh.toml";
+
+pub async fn init_cloud_hypervisor_sandboxer(
```

Comment:
why is this init function needed? maybe move it to cloud_hypervisor folder?

Reply:
I tried moving the initialization functions for `qemu`,`cloud_hypervisor`, and `stratovirt` in `lib.rs` to their own folders.